source/chapters/user_interface.rst

```diff
@@ -666,15 +693,34 @@ Equalizers and Gain Knobs
    EQ Controls
 
 **Gain Knob**
-  Above these knobs, the gain knob allows you to adjust the gain of the deck. In
-  general, you should adjust this knob so that at full-volume the deck's audio
-  is just at the peak of the center VU meters. This is so you can achieve the
-  widest dynamic range for your track.
+  Above these knobs, the gain knob allows you to adjust the gain of the deck. 
+  Use this to compensate for the differences in recording levels between tracks.
+  In general, you should adjust this knob so that the track's :term:`level meter`
+  stays around the top of the green region with the loudest parts of the track
+  (the transients) briefly going into the yellow region. **Do NOT turn the
+  Gain Knob up so much that the level meter is in the red region. At this point
+  the track is clipping, which sounds bad and could damage equipment.**
+  
+  .. seealso:: For a more in depth explanation of how to set your gains, see
+               :ref:`Setting your levels properly (gain staging) <djing-gain-staging>`.
```

Comment:
It would be nice, if we have a note about a recommended recording and broadcasting level:
"Recommendation:  "All gain Knobs at unity ReplayGain at -16 LUFS"